app/templates/teachers/teacher-student-view.jade

```diff
@@ -10,7 +10,9 @@ block content
   else
     .container
       +breadcrumbs
-      if view.classroom && view.classroom.get('ownerID') === me.id || me.isAdmin()
+      if !view.classroom || view.classroom.get('ownerID') !== me.id || !me.isAdmin()
```

Comment:
Would this work for an admin spying on a teacher?
I think the condition should be `!view.classroom || (view.classroom.get('ownerID') !== me.id && !me.isAdmin())`

Reply:
Also, you could create a utility function in the TeacherStudentView.coffee that would check these conditions to determine if the page is accessible or not, to make this cleaner. (eg: `canViewStudentProfile` / something like this)

Reply:
Thank you for catching this! Somehow I managed to read this as three `&&`'s.